docker-compose.yml

```diff
@@ -283,6 +283,12 @@ services:
   snuba-subscription-consumer-transactions:
     <<: *snuba_defaults
     command: subscriptions-scheduler-executor --dataset transactions --entity transactions --auto-offset-reset=latest --no-strict-offset-reset --consumer-group=snuba-transactions-subscriptions-consumers --followed-consumer-group=transactions_group --delay-seconds=60 --schedule-ttl=60 --stale-threshold-seconds=900
+  snuba-profiling-profiles-consumer:
+    <<: *snuba_defaults
+    command: consumer --storage profiles --auto-offset-reset=latest --max-batch-time-ms 1000 --no-strict-offset-reset
+  snuba-profiling-functions-consumer:
+    <<: *snuba_defaults
+    command: consumer --storage functions --auto-offset-reset=latest --max-batch-time-ms 1000 --no-strict-offset-reset
```

Comment:
Should these be added to the list of sentry image's depends_on field?

Here: https://github.com/getsentry/self-hosted/blob/8aa571cb758406eccb7aefe13b3b7ebf1a34e5c2/docker-compose.yml#L39-L50

Reply:
I added `vroom` as a `depends_on` since `web` needs to POST to `vroom`, but then the consumers only depends on `kafka` technically. I think it's good like this.